pydicom/tests/test_handler_util.py

```diff
@@ -941,7 +941,7 @@ def test_neither_ds_nor_palette_raises(self):
         with pytest.raises(ValueError, match=msg):
             apply_color_lut(ds.pixel_array)
 
-    def test_palette_unknown_raises(self):
+    def test_palette_unknown_raises(self, disable_value_validation):
         """Test using an unknown `palette` raise an exception."""
```

Comment:
Disabled value validation in all tests that would cause warnings to avoid having them in the test log.